packages/storybook/stories/va-text-input.stories.jsx

```diff
@@ -9,24 +9,46 @@ export default {
 };
 
 const defaultArgs = {
-  name: 'my-input',
-  label: 'My input',
-  autocomplete: false,
+  'name': 'my-input',
+  'label': 'My input',
+  'autocomplete': false,
   // Fun fact: This should really be false, but that passes
   // enable-analytics="false" to the web component, which correctly interprets
   // this as a truthy value. See also:
   // https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attribute
-  'enable-analytics': null,
+  'enable-analytics': false,
```

Comment:
given the note above, should 'enable-analytics' and 'required' stay `null`?

Reply:
The comments are correct in that a falsy value should not be displayed on the DOM at all. The reason the prop was updated to false is because Storybook correctly displays how it should be used when clicking on 'Show code' for a component. 

Note: If you inspect the DOM, it shows up as `<va-component enable-analytics="false">`. This is wrong and it's an issue affecting Stencil.

https://github.com/ionic-team/stencil-ds-output-targets/issues/123
https://github.com/ionic-team/stencil-ds-output-targets/pull/149/files

All this change is doing is making sure `enable-analytics` does not show up as `enable-analytics={null}` when false. It is correctly omitted when given a value of false.

Reply:
Checked locally and confirmed value does not display on element when false inside of Storybook "Show Code"

Reply:
Ok, that all makes sense. But, if we are going to have them as 'false' here, we should probably update the note so that it better represents the current situation. Is this something that FE devs will need to be aware of?

Reply:
I've removed the comments since they no longer apply to how it's being used here. https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attribute is still relevant and devs should be aware of it, but I'm not sure if we have to take action because it's general HTML knowledge. What do you think?

We can and should verify Storybook's "Show Code" examples accurately represent how a developer should use each component. This means that when creating or updating a Storybook story, verify the component is receiving the correct data and data types for props. In the case of a boolean prop with a value of false, verify it is omitted when looking at the code in "Show Code".

Using the alert web component with a value of false for the boolean prop named background-only:
<img width="1019" alt="Screen Shot 2021-12-06 at 17 02 35" src="https://user-images.githubusercontent.com/36863582/144930813-872ebd79-aa56-4be3-9113-4c4ac99e6c47.png">

Using the alert web component with a value of true for the boolean prop named background-only:
<img width="1014" alt="Screen Shot 2021-12-06 at 17 02 49" src="https://user-images.githubusercontent.com/36863582/144931120-4ea26d0f-fdf0-4957-be56-4046c9727677.png">

Reply:
I am good with this as long as it is clear how developers should use this component. Thank you!